jekyll/_cci2/testing-orbs.md

```diff
@@ -8,178 +8,170 @@ order: 1
+Like any software, especially open source software expecting community contributions, proper testing is important. Because orbs are ultimately YAML, it may seem difficult to properly test your orbs, but, with the Orb Development Kit it is easy to implement a full range of robust tests for your orb.
 
-* Schema Validation - this can be done with a single CLI command and checks if the orb is well-formed YAML and conforms to the orb schema.
-* Expansion Testing - this can be done by scripting the CircleCI CLI and tests whether the elements of the orb generate the configuration you intended when processing configuration containing those elements.
-* Runtime Testing - this requires setting up separate tests and running them inside a CircleCI build.
-* Integration Testing - this is likely only needed for fairly advanced orbs or orbs designed specifically as public, stable interfaces to 3rd-party services. Doing orb integration tests requires a custom build and your own external testing environments for whatever systems you are integration with.
 
-The sections below provide details on each of these techniques for each level of orb testing.
+## Validation
 
-### Schema Validation
+The most basic form of "testing" for orbs is basic configuration validation, and code linting. When an orb is packed and published it must both be valid YAML, and valid CircleCI syntax. Both of these checks can be performed manually locally and are also automatically in the Orb Development Kit provided config file at `.circleci/config.yml`.
 
-To test whether an orb is valid YAML and is well-formed according to the schema, use `circleci orb validate` with the CircleCI CLI.
+When you first make a commit to your orb repository, this will trigger the [test-pack](https://github.com/CircleCI-Public/Orb-Project-Template/blob/43712ad367f2f3b06b2ae46e43ddf70bd3d83222/.circleci/config.yml#L40) workflow, which contains several jobs related to validating and testing our orb.
 
-#### Example
+Learn more about the included config file on [Orb Publishing Process]({site.baseurl}}/2.0/creating-orbs).
 
-Given an orb with source at `./src/orb.yml` you can run `circleci orb validate ./src/orb.yml` to receive feedback on whether the orb is valid and will pass through config processing. If there is an error, you will receive the first schema validation error encountered. Alternatively, you can pass STDIN rather than a file path.
+### YAML Lint
 
-For example, equivalent to the previous example you can run `cat ./src/orb.yml | circleci orb validate -`
+The first job listed within the workflow, `orb-tools/lint` is from our [orb tools orb](https://circleci.com/orbs/registry/orb/circleci/orb-tools), another major component of the Orb Development Kit. The `orb-tools/lint` job is responsible for basic YAML linting. You can modify the linting rules or other settings via the [job's parameters, which are listed on the orb registry](https://circleci.com/orbs/registry/orb/circleci/orb-tools#jobs-lint).
```

Comment:
weirdly this link https://circleci.com/orbs/registry/orb/circleci/orb-tools#jobs-lint is not taking me to the lint job, but the top of the registry page?

Reply:
Looks to be a bug with the registry. I think i remember this being discussed recently though. CC @lokst